.github/workflows/github-actions-on-label-create.yml

```diff
@@ -1,33 +1,69 @@
 name: Labelled Ready to Sync Public
+
 on:
   pull_request:
     types: [labeled]
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
+
+env:
+  PRIVATE_OWNER: ${{ github.event.pull_request.head.repo.owner.login }}
+  PRIVATE_REPO: ${{ github.event.pull_request.head.repo.name }}
+  PRIVATE_BRANCH: ${{ github.event.pull_request.head.ref }}
+  # Staging repository
+  STAGING_OWNER: The-OpenROAD-Project-staging
+  STAGING_REPO: OpenROAD
+  STAGING_BRANCH: ${{ github.event.pull_request.head.ref }}
+  # Upstream repository
+  UPSTREAM_OWNER: The-OpenROAD-Project
+  #UPSTREAM_REPO: ${{ github.event.pull_request.base.repo.name }}
+  UPSTREAM_BRANCH: master
 
 jobs:
-  label:
+  Push-To-Staging:
+    name: Push to staging
     runs-on: ubuntu-latest
-    if: github.repository_owner == 'The-OpenROAD-Project-private' && github.event.label.name == 'Ready To Sync Public'
+
+    permissions:
+      # Read-only access so we don't accidentally try to push to *this* repository.
+      contents: read
+      # Pull request write access, so we can remove the label.
+      issues: write
+      pull-requests: write
+      # Deployment write access, so we can add the nice deployment info.
+      deployments: write
+
+    # Only run on the private repository.
+    if: github.event.repository.private && github.event.label.name == 'Ready To Sync Public'
```

Comment:
This line replace the check for the explicit org. I think that is safe since the repos are only private in the -private org.

Reply:
Now with the staging repository configured using `STAGING_OWNER` secret, anyone can use this style of Private->Staging->Public workflow.